lib/rihanna/job.ex

```diff
@@ -349,6 +372,21 @@ defmodule Rihanna.Job do
       )
   end
 
+  @doc """
+  Checks if a job implements `priority` callback and runs it.
+
+  A lower value means a higher job priority. Has a default of 20.
```

Comment:
Why is the column default 50?

Reply:
I'll change the comment. The default of 50 is just a big number so jobs run at a very low priority by default.